pkgs/development/python-modules/pyside2/default.nix

```diff
@@ -13,16 +13,40 @@
 }:
 stdenv.mkDerivation rec {
   pname = "pyside2";
-  version = "5.15.11";
+  version = "5.15.14";
 
   src = fetchurl {
-    url = "https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-${version}-src/pyside-setup-opensource-src-${version}.tar.xz";
-    sha256 = "sha256-2lZ807eFTSegtK/j6J3osvmLem1XOTvlbx/BP3cPryk=";
+    url = "https://mirrors.ukfast.co.uk/sites/qt.io/official_releases/QtForPython/pyside2/PySide2-${version}-src/pyside-setup-opensource-src-${version}.tar.xz";
```

Comment:
is it feasable to hardcode a (UK) mirror?

Reply:
Ah I will swap this back. I had it changed while the default download server was broken.